src/gui/qgsnewgeopackagelayerdialog.cpp

```diff
@@ -473,3 +475,7 @@ bool QgsNewGeoPackageLayerDialog::apply()
   return false;
 }
 
+void QgsNewGeoPackageLayerDialog::helpRequest()
+{
+  QgsHelp::openHelp( "managing_data_source/opening_data.html#loading-a-layer-from-a-file" );
```

Comment:
The right link is managing_data_source/create_layers.html#creating-a-new-geopackage-layer. It's not yet available in the doc though (pending [PR #2000](https://github.com/qgis/QGIS-Documentation/pull/2000)).